index.html

```diff
@@ -11867,7 +11867,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>User agents MUST either prune <code>aria-hidden</code> nodes from the accessibility tree, or mark all <code>aria-hidden</code> nodes as hidden.</p>
 				<p>Authors MAY use <code>aria-hidden="false"</code> within an <code>aria-hidden="true"</code> subtree to unhide the visibly rendered content within the <code>aria-hidden="false"</code> subtree.</p>
 				<p class="note">This is a change from ARIA 1.2 and earlier versions in several ways: 1) <code>aria-hidden="false"</code> no longer un-hides content that is also visible in CSS, as there are other techniques to present content only to AT users that works more consistently (e.g. positioning offscreen), and 2) <code>aria-hidden="false"</code> previously was a noop on visible content, and 3) <code>aria-hidden="false"</code> previously operated only on a single element. However, the <code>aria-hidden="false"</code> behavior of previous versions was never consistently implemented, understood or commonly used.</p>
-				<p>Authors MUST NOT place focusable elements within <code>aria-hidden</code> content. This is because the AT will have no information about the content, so it cannot be presented to the user with AT. User agents MUST repair this situation by ignoring the <code>aria-hidden</code> attribute on the ancestor of the focused element.</p>
+				<p>Authors MUST NOT place focusable elements within <code>aria-hidden</code> content. This is because the AT will have no information about the content, so it cannot be presented to the user with AT. User agents MUST repair this situation by ignoring the <code>aria-hidden</code> attribute on the ancestor of the focused element. When a repair is made, user agents should inform authors through developer tools (if available).</p>
```

Comment:
uppercase to should.
remove awkward/recursive second mention of AT from the "this is because the at..." sentence.

```suggestion
				<p>Authors MUST NOT place focusable elements within <code>aria-hidden</code> content. This is because the AT will have no information about the content, so it cannot be presented to the user. User agents MUST repair this situation by ignoring the <code>aria-hidden</code> attribute on the ancestor of the focused element. When a repair is made, user agents SHOULD inform authors through developer tools (if available).</p>
```